src/serial.rs

```diff
@@ -734,6 +734,32 @@ usart! {
     USART5: (usart5, apb1enr, usart5en, apb1_clk, Serial5Ext),
 }
 
+impl Serial<LPUART1> {
+    pub fn use_lse(&self, rcc: &mut Rcc, _: &LSE) {
```

Comment:
Is the `&self` a deliberate choice? I think it could be very surprising, if someone held a shared reference to `Serial` and somehow the configuration suddenly changed (especially since that might change baud rate with the current implementation). Unless there's a good reason not to, I'd prefer `&mut self`.

Reply:
@hannobraun Good catch :)